SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -669,6 +669,11 @@ void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningW
 		return;
 	}
 
+	if (!IsServer())
```

Comment:
made this change across both PRs now

Reply:
This is going in elsewhere - worth removing to help conflicts